frontend/server/minio-helper.ts

```diff
@@ -54,24 +56,80 @@ export async function createMinioClient(config: MinioClientOptionsWithOptionalSe
   return new MinioClient(config as MinioClientOptions);
 }
 
-export function getTarObjectAsString({ bucket, key, client }: MinioRequestConfig) {
-  return new Promise<string>(async (resolve, reject) => {
-    try {
-      const stream = await getObjectStream({ bucket, key, client });
-      let contents = '';
-      // TODO: fix tar.Parse typing problem
-      stream.pipe(new (tar.Parse as any)()).on('entry', (entry: Stream) => {
-        entry.on('data', buffer => (contents += buffer.toString()));
-      });
-      stream.on('end', () => {
-        resolve(contents);
-      });
-    } catch (err) {
-      reject(err);
-    }
+/**
+ * Checks the magic number of a buffer to see if the mime type is a uncompressed
```

Comment:
Can you put a reference link here?

Reply:
done
Add `https://www.gnu.org/software/tar/manual/html_node/Standard.html` to comment.